### PR TITLE
Bump parent POM from 1.65 to 1.69

### PR DESCRIPTION
**Pull request by basil**
Like #289 but with SpotBugs warnings suppressed and some Enforcer changes.

### CODE REVIEW

**Review by timja — APPROVED** [review]

File: jelly/pom.xml [hunk 1]
```diff
@@ -10,6 +10,15 @@
   <name>Stapler Jelly module</name>
   <description>Jelly binding for Stapler</description>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>commons-lang</groupId>
+        <artifactId>commons-lang</artifactId>
+        <version>2.5</version>
```
Comment (jglick) [inline jelly/pom.xml:18]: Maybe better to `<exclude>` it from wherever it was being improperly picked up?

**Review by jglick — APPROVED** [review]

File: jelly/pom.xml (hunk 1 above)
Comment (basil) [reply jelly/pom.xml:18]: I pasted the error in the commit message. So where do you think it is being improperly picked up?

Comment (jglick) [reply jelly/pom.xml:18]: Looks like both `htmlunit` and `ezmorph`, with `json-lib` asking for the newest version. Not worth bothering over. (When there is just one dep trail which picks up an old lib, and unfortunately that happens to be the one preferred by Maven’s silly “nearest” algorithm, then it is easier to exclude that trail than to maintain an explicit version number.)

Comment (basil) [reply jelly/pom.xml:18]: > When there is just one dep trail which picks up an old lib, and unfortunately that happens to be the one preferred by Maven’s silly “nearest” algorithm, then it is easier to exclude that trail than to maintain an explicit version number

Yeah, I agree. It gets particularly perverse when Dependabot then starts trying to update that number in the `<dependencyManagement>` section.

**Pull request merged**